### PR TITLE
Added templates for MozFest German, French, and Spanish proposal confirmation email

### LINKED ISSUE #60: Add templates for MozFest German and French proposal confirmation email
Context: https://github.com/mozilla/mozillafestival.org/issues/602
Translations are in here: https://l10n.mozilla-community.org/langchecker/?website=6&file=emails/2017/mozfest_call.lang&action=translate&show

English one is already in place: https://github.com/mozilla/webmaker-mailroom/pull/59 We need to add German and French as well. I will submit a PR.

/cc @cadecairos @gideonthomas

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #60

/cc @TheoChevalier

### CODE REVIEW

**Comment by mmmavis** [comment]
@cadecairos do we have to add something like `<meta charset="utf-8">` to `header.html` for these characters to show properly?

File: templates/mozfest_session_proposal_2017/index.html [hunk 1]
```diff
@@ -16,6 +16,6 @@
   <p>Best,
   <br />The MozFest Team</p>
 
-  <p>PS - MozFest curation takes place in the open. This means some information, like your name and details about your proposed session, are visible in <a href="{{ github_issue_url }}">your Github ticket</a>. If you would like this information removed for privacy reasons, please let us know at  <a href="mailto:festival@mozilla.org">festival@mozilla.org</a></p>
+  <p>PS - MozFest curation takes place in the open. This means some information, like your name and details about your proposed session, are visible in <a href="{{ github_issue_url }}">your Github ticket</a>. If you would like this information removed for privacy reasons, please let us know at <a href="mailto:festival@mozilla.org">festival@mozilla.org</a></p>
```
Comment (mmmavis) [inline templates/mozfest_session_proposal_2017/index.html]: removed extra space

**Comment by mmmavis** [comment]
Ping @cadecairos @gideonthomas Could we have this reviewed today? A few MozFest L10N updates need to go live by June 15 so we should have everything ready soon.

**Comment by cadecairos** [comment]
@mmmavis I'll take care of this for you asap

**Comment by mmmavis** [comment]
@cadecairos lovely! Do you have ETA when these changes can be live? I'll need to update code on MozFest site so non-English proposals get confirmation email in the language the proposal was written in.

**Comment by cadecairos** [comment]
@mmmavis I'll merge and tag this after the staff call, then get the version number in sawmill bumped and deployed.

**Pull request merged**